library-best-practices/glossary.md

```diff
@@ -101,32 +101,35 @@ Java Dependency Glossary
   In general, there can be multiple (parallel) edges between two nodes when
   a class references two or more methods and fields of another class.
   Self-loops (references between the same class) are possible and
-  common. However, it is safe to omit such references during static linkage checks,
-  because the Java compiler ensures that the self-loop references are linkage compatible.
+  common. However, it is safe to omit such references when checking the reachability
+  to static linkage errors, because the Java compiler ensures that the self-loop
+  references are linkage compatible.
 
-- **A reference**: in the class usage graph, the relationship between two 
-  classes is either a _class reference_, _method reference_ or _field reference_.
-
-  A _method reference_ indicates that the source class invokes a (static or
-  non-static) method of the destination class.
-
-  A _field reference_ indicates that the source class accesses a (static or
-  non-static) field of the destination class.
+- **Reference**: a relationship from a class to another class on how the former
```

Comment:
How about: "A relationship from one class to another class such that the first class requires the presence the second class in a particular form in order to function."

Reply:
Updated.
Still this part is not 100% sure though >  "in a particular form in order to function"